locales/ca/generate.json

```diff
@@ -0,0 +1,52 @@
+{
+   "meta": {
+      "description": "Genera llistes de música a Spotify basat en les cançons que estàn tocant en aquell tour, utilizant la API de Setlist.fm",
+      "keywords": "Spotify, Setlist.fm, Generador de llistes, Festivals, Música, Concerts",
+      "title": "Generar | Festwrap"
+   },
+   "steps": {
+      "navigation": {
+         "finish": "Finalitzar",
+         "generate": "Generar",
+         "next": "Següent",
+         "previous": "Anterior"
+      },
+      "step1": {
+         "description": "Selecciona per crear una nova llista o utilitza una llista ja existent en el teu compte",
+         "form": {
+            "createNewPlaylist": {
+               "description": "Utilitza aquesta opció per crear una nova llista de reproducció a Spotify",
+               "giveAName": "Dóna-li un nom a la llista",
+               "namePlaceholder": "Nom de la llista",
+               "privatePlaylist": {
+                  "description": "La llista de reproducció serà privada i només tu podràs veure-la",
+                  "title": "Llista privada"
+               },
+               "title": "Crear una nova llista"
+            },
+            "useExistingPlaylist": {
+               "description": "Utilitza aquesta opció per afegir cançons a una llista de reproducció ja existent",
+               "selectPlaylist": "Selecciona una llista de reproducció",
+               "title": "Utilitzar una llista existent"
+            }
+         },
+         "title": "Tria una opció per començar"
+      },
+      "step2": {
+         "description": "Busca els artistes que vols afegir a la llista de reproducció",
```

Comment:
I would replace "busca" with "afegeix" in this second step. The ultimate goal is to add artists, searching is just the means to it.